src/components/StandardsGallery/index.tsx

```diff
@@ -133,42 +133,44 @@ const STANDARDS = [
 
 function StandardCard({ link, title, name, text }) {
   return (
-    <div className={`col col--3 margin-bottom--lg ${styles.legoCard}`}>
-      <div>
-        <Link
-          to={link}
-          style={{
-            borderWidth: '1px',
-          }}
-        >
-          <h3 className="flex items-center font-jakarta">
-            <div>{title}</div>
-          </h3>
-          <h4 className="flex items-center font-jakarta">
-            <div className={styles.standardName}>{name}</div>
-          </h4>
-          <p className="mb-0 text-sm text-zinc-400">{text}</p>
-        </Link>
-      </div>
-    </div>
+    <Link
+      className={`col col--4 margin-bottom--lg ${styles.legoCard}`}
+      to={link}
+      style={{
+        borderWidth: '1px',
+      }}
+    >
+      <h2 className="flex items-center font-jakarta">
+        <div>{title}</div>
```

Comment:
We usually avoid to put block elements like this div inside a heading. You probably want to use a span or something inline.

Check

https://stackoverflow.com/questions/66084722/does-html5-allow-a-block-level-element-like-div-inside-a-heading-like-h1

Reply:
Here I think this div is useless anyway.

Reply:
true I don't know why it's there. Probably a mistake